dragonfly/implementations/daq_run_interface.py

```diff
@@ -128,7 +142,7 @@ def _do_postrun_gets(self):
             self._run_snapshot['LOGS'].update(these_snaps)
 
     def determine_RF_ROI(self):
-        raise core.exceptions.DriplineMethodNotSupportedError('subclass must implement RF ROI determination')
+        logger.warning('subclass must implement RF ROI determination; skipped')
```

Comment:
I think the determine_RF_ROI is required in general and should not just send a warning in production mode.
I don't see how testing the DAQInterface makes sense on insectarium from a production mode testing, since no DAQ is actually here. 
Also nothing prevent you (when testing on insectarium) from making a dummy determine RF ROI when testing on insectarium (dummy could mean an empty method or commenting the actual commands sent to get the parameters and replace the values start_freq... y dummy values).